docs/MYSQL_WC.md

```diff
@@ -0,0 +1,52 @@
+
+### MySQL Connection URI
+
+A Kart repository with a MySQL working copy needs to be configured with a `mysql://` connection URI. This URI contains how to connect to the server, and the name of the database that should be managed as a working copy by this Kart repository.
+
+A connection URL would generally have the following format:
+
+`mysql://[user[:password]@][host][:port]`
+
+Since Kart also requires the database to be specified up front, Kart needs a connection URL in the following format:
+
+`mysql://[user[:password]@][host][:port]/dbname`
+
+For example, a Kart repo called `airport` might have a URL like the following:
+
+`mysql://kart_user:password@localhost:1433/airport_kart`
+
+To configure a Kart repository to use a particular MySQL database as its working copy, specify the `--workingcopy` flag when creating the repository, for example:
+
+`kart init --workingcopy=mysql://... --import=...`
+
+The database that Kart is given to manage should be either non-existent or empty at the time Kart is configured, but the server should already be running.
+
+The database user needs to have full rights to modify objects in the specified database. (eg: via `GRANT ALL PRIVILEGES ON airport_kart.* TO kart_user; FLUSH PRIVILEGES;`).
+
+### MySQL limitations
+
+Most geospatial data can be converted to MySQL format without losing any fidelity, but it does have the following limitations.
+
+#### Three and four dimensional geometries
+
+Geometries in MySQL are always two-dimensional (meaning they have an X and a Y co-ordinate, or a longitude and a latitude co-ordinate). Three- or four-dimensional geometries, with Z (altitude) or M (measure) co-ordinates, are not supported in MySQL. As a result, Kart datasets containing three- and four-dimensional geometries cannot currently be checked out as MySQL working copies. More documentation will be added here if or when any workarounds for this MySQL limitation are added to Kart.
+
+#### Approximated types
+
+There is one type that Kart supports that has no MySQL equivalent - the `interval`. This type is approximated as `TEXT` in the My Server working copy. See [APPROXIMATED_TYPES](APPROXIMATED_TYPES.md) for more information.
+
+#### CRS definitions
+
+MySQL comes pre-installed with thousands of the standard EPSG & ESRI coordinate reference system definitions. Currently, only the CRS definitions that are already in your MySQL installation are supported - Kart will not create definitions in MySQL to match the custom definitions attached to your Kart datasets. More documentation will be added here when this is supported.
```

Comment:
(i might be wrong but) I don't think any ESRI CRS definitions are in MySQL (actually I think they're not in postgis either  (?))
https://dev.mysql.com/worklog/task/?id=8579

It also looks like they added all the CRSes in 8.0, which might be a good reason to note why we don't support earlier versions.

```suggestion
MySQL comes pre-installed with thousands of standard EPSG coordinate reference system definitions. Currently, only the CRS definitions that are already in your MySQL installation are supported - Kart will not create definitions in MySQL to match the custom definitions attached to your Kart datasets. More documentation will be added here when this is supported.
```